docker/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Ruun some setup tasks before starting the main application
+echo "Setting up the environment..."
+
+# Print the directory structure for debugging purposes
+echo "File structure:"
+find /app
```

Comment:
@vasylboliuk you could use `tree` which has way more functionality and describes project structure better